packages/ui-components/src/components/toggle-tip/toggle-tip.scss

```diff
@@ -0,0 +1,44 @@
+@import '../../assets/styles/globals';
+
+:host {
+	/**
+	 * @prop --button-icon-width: Button's icon width.
+	 * @prop --button-icon-height: Button's icon height.
+	 */
+
+	--button-icon-width: 16px;
+	--button-icon-height: 16px;
+}
+
+kv-icon {
+	--icon-width: var(--button-icon-width);
```

Comment:
No need for this. Export the `kv-icon` part so it can be styled by the toggle parent.